kratos/mpi/utilities/model_part_communicator_utilities.h

```diff
@@ -0,0 +1,80 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main author:     Jordi Cotela
+//
+
+#ifndef KRATOS_MODEL_PART_COMMUNICATOR_UTILITIES_H_INCLUDED
+#define	KRATOS_MODEL_PART_COMMUNICATOR_UTILITIES_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "mpi/includes/mpi_communicator.h"
+
+namespace Kratos
+{
+
+///@addtogroup MPICore
+///@{
+
+///@name Kratos Classes
+///@{
+
+/// Utilitiy class for ModelPart::Comunicator management in an MPI context.
+class ModelPartCommunicatorUtilities
```

Comment:
I think a namespace would fit better here

Reply:
I agree, but I kept the class structure to export it to python using our standard approach. I am not sure about how we should name the pybind wrapper for a method on a namespace.